docs/DEEP_DIVE.md

```diff
@@ -26,11 +32,11 @@ scripts/install.sh
 
 ## Simulate non-pristine lab conditions
 
-You are ready to rock! To simulate non-pristine lab conditions (tiger beetles thrive in harsh environments with noisy neighbors) turn up your favorite album, or if you're looking for something new we suggest Noel Gallagher's High Flying Birds.
+You are ready to rock! To simulate non-pristine lab conditions (tiger beetles thrive in harsh environments with noisy neighbors), turn up your favorite album, or if you're looking for something new, we suggest Noel Gallagher's High Flying Birds.
 
 Be aware that super non-pristine lab conditions such as an active video call might cause cache pollution and drop throughput. But rock 'n roll is generally fine.
 
-## Benchmark!
+## Benchmark
```

Comment:
```suggestion
## Benchmark!
```

Reply:
Sorry about that one, I should disable auto-lint for markdown on my editor